src/js/app.js

```diff
@@ -1,52 +1,18 @@
-import { 
-    buildGameTable
-} from "./gameBuilder.js";
-import { removeGameTable } from "./gameState";
+import { buildGameTable } from "./gameBuilder.js";
 import { stopTimer, resetTimeCounter } from "./timer";
 
-var level;
-setLevel();
+const restartElement = document.getElementById("restart");
+const levelElement = document.getElementById('level');
 
-export var table = buildGameTable(level);
+export let table = {};
 
-document.getElementById("restart").addEventListener("click", restartGame);
+restartElement.addEventListener("click", restartGame);
+levelElement.addEventListener("change", restartGame);
 
+restartGame();
 
 function restartGame() {
-    setLevel();
-    removeGameTable();
-    table = buildGameTable(level);
+    table = buildGameTable(levelElement.value.split(',').map(item => +item));
     stopTimer();
```

Comment:
We directly pass the value from the `<select>` to the function. This way we can change the levels in one place (the index.html)